github_review_helper_suite_test.go

```diff
@@ -0,0 +1,17 @@
+package main_test
+
+import (
+	"io/ioutil"
+	"log"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
Can you specify git commit dep in the import? I don't like open-ended master deps.